phygnn/model_interfaces/phygnn_model.py

```diff
@@ -123,20 +127,19 @@ def train_model(self, features, labels, p, n_batch=16, n_epoch=10,
         Parameters
         ----------
         features : np.ndarray | pd.DataFrame
-            Feature data in a 2D array or DataFrame. If this is a DataFrame,
+            Feature data in a >=2D array or DataFrame. If this is a DataFrame,
             the index is ignored, the columns are used with self.feature_names,
             and the df is converted into a numpy array for batching and passing
-            to the training algorithm.
+            to the training algorithm. A 2D input should have the shape:
+            (n_observations, n_features). A 3D input should have the shape:
+            (n_observations, n_timesteps, n_features). 4D inputs have not been
+            tested and should be used with caution.
         labels : np.ndarray | pd.DataFrame
-            Known output data in a 2D array or DataFrame. If this is a
-            DataFrame, the index is ignored, the columns are used with
-            self.output_names, and the df is converted into a numpy array for
-            batching and passing to the training algorithm.
+            Known output data in a 2D array or DataFrame.
+            Same dimension rules as x.
```

Comment:
should this and p be same dimension rules as "features"?

Reply:
ah yes. just copied the doc string from phygnn